src/settings_type.h

```diff
@@ -108,6 +108,7 @@ struct GUISettings {
 	uint8  window_soft_limit;                ///< soft limit of maximum number of non-stickied non-vital windows (0 = no limit)
 	ZoomLevel zoom_min;                      ///< minimum zoom out level
 	ZoomLevel zoom_max;                      ///< maximum zoom out level
+	ZoomLevel sprite_zoom_min;               ///< maximum GRF-provided zoom-level to use, forces scaling for further zoom levels
```

Comment:
as talked about on IRC, I do not understand this comment. Mentioning GRF feels confusing, but mostly I think the english text captures the essence of this setting a lot more.

Reply:
Updated, I seem to be having a bit of a brain-mashed week so not sure how great the description is but hopefully it's a bit clearer.